src/tests/multihost/alltests/test_automount.py

```diff
@@ -355,6 +355,7 @@ def test_009_maps_after_coming_online(self, multihost, add_nisobject):
         :title: IDM-SSSD-TC: ldap-Provider Automount: Without eisting cache
```

Comment:
I thought we will not use this old format for test case name? this was fine
https://github.com/SSSD/sssd/pull/5478/files#diff-fd19376c43f72875d5fcec276b642958bf7485d03fc86c4484748b071e16fee5R351